### PR TITLE
Feature/task4

### PR DESCRIPTION
**Pull request by Kolenchitssi**

### CODE REVIEW

File: tasks/task4.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/* Task 1 */
+function createObject(obj) {
+  const obj1 = obj;
```
Comment (irina-chernik) [inline tasks/task4.js]: Make function universal.
Not only for getTestField method

File: tasks/task4.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      return resultObj.item;
+    };
+
+    resultObj.read = () => resultObj.item;
+    return resultObj;
+  };
+
+  this.getBy = (func) => {
+    const resultObjItems = {};
+    resultObjItems.arrItems = this.collection.filter(func);
```
Comment (irina-chernik) [inline tasks/task4.js]: result  object contains extra property arrItems that is not supposed to be here
same for result object for get with item property

File: tasks/task4.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    resultObjItems.update = function (funcUpdate) {
+      resultObjItems.arrItems.forEach((item, index) => {
```
Comment (irina-chernik) [inline tasks/task4.js]: for short one line expressions in arrow function no need to use {}, use shortened version
same goes for functions above (update as well)

**Review by irina-chernik — CHANGES_REQUESTED** [review]

File: tasks/task4.js (hunk 1 above)
Comment (Kolenchitssi) [reply tasks/task4.js]: completed

File: tasks/task4.js (hunk 3 above)
Comment (Kolenchitssi) [reply tasks/task4.js]: completed

File: tasks/task4.js (hunk 2 above)
Comment (Kolenchitssi) [reply tasks/task4.js]: completed

**Review by irina-chernik — APPROVED** [review]

**Pull request merged**